WooCommerce/Woo Watch App/Orders/OrdersListView.swift

```diff
@@ -114,17 +114,14 @@ struct OrdersListView: View {
     /// Data: List with live order content.
     ///
     @ViewBuilder private func dataView(orders: [Order]) -> some View {
-        List() {
+        List {
             ForEach(orders, id: \.number) { order in
-                NavigationLink(value: order) {
+                NavigationLink(destination: OrderDetailView(order: order)) {
```

Comment:
The non-closure version of `NavigationLink` is deprecated from 17.5, but this version is available instead:

```suggestion
                NavigationLink {
                    OrderDetailView(order: order)
                } label: {
```